initializer/initializer.go

```diff
@@ -157,25 +157,13 @@ var (
 	metricsWorkPool, readWorkPool      *workpool.WorkPool
 )
 
-func Initialize(
-	logger lager.Logger,
-	config ExecutorConfig,
-	cellID string,
-	zone string,
-	rootFSes map[string]string,
-	metronClient loggingclient.IngressClient,
+func Initialize(logger lager.Logger, config ExecutorConfig, cellID, zone string,
+	rootFSes []string, metronClient loggingclient.IngressClient,
 	clock clock.Clock,
-) (
-	executor.Client,
-	*containermetrics.StatsReporter,
-	grouper.Members,
-	error,
-) {
-
+) (executor.Client, *containermetrics.StatsReporter, grouper.Members, error) {
 	var gardenHealthcheckRootFS string
-	for _, rootFSPath := range rootFSes {
-		gardenHealthcheckRootFS = rootFSPath
-		break
+	if len(rootFSes) > 0 {
+		gardenHealthcheckRootFS = rootFSes[len(rootFSes)-1]
```

Comment:
This code changes the behavior (I think) from before. 

Before it looks like gardenHealthcheckRootFS was set to the first rootFSPath. Now it looks like it is being set to the last last rootFSPath. 

❓ What is the intention behind this change? Is there a test change to go with it?